woori-db/src/actors/state.rs

```diff
@@ -75,8 +75,10 @@ impl Handler<PreviousRegistry> for Executor {
                 .ok_or(Error::FailedToParseRegistry)?
                 .to_owned();
             let state = &state[..(state.len() - 1)];
+            println!("{:?}", state);
```

Comment:
This `println` should stay here? Was it for debugging?

Reply:
2f36dec